quickjs-libc.c

```diff
@@ -94,6 +94,16 @@ extern char **environ;
 #include "list.h"
 #include "quickjs-libc.h"
 
+#ifdef _WIN32
+#ifndef QJS_NATIVE_MODULE_SUFFIX
+#define QJS_NATIVE_MODULE_SUFFIX ".dll"
+#endif
+#else
+#ifndef QJS_NATIVE_MODULE_SUFFIX
+#define QJS_NATIVE_MODULE_SUFFIX ".so"
+#endif
+#endif
```

Comment:
Why define this conditionally? And why not more succinctly like this?
```suggestion
#ifndef QJS_NATIVE_MODULE_SUFFIX
#ifdef _WIN32
#define QJS_NATIVE_MODULE_SUFFIX ".dll"
#else
#define QJS_NATIVE_MODULE_SUFFIX ".so"
#endif
#endif
```